apps/scan/backend/src/scanners/pdi/state_machine.ts

```diff
@@ -19,52 +29,33 @@ import {
   sendParent,
   spawn,
 } from 'xstate';
-import { v4 as uuid } from 'uuid';
-import {
-  SheetInterpretation,
-  SheetOf,
-  ballotPaperDimensions,
-  mapSheet,
-} from '@votingworks/types';
-import { join } from 'path';
-import { writeImageData } from '@votingworks/image-utils';
-import { BaseLogger, LogEventId, LogLine } from '@votingworks/logging';
-import { UsbDrive } from '@votingworks/usb-drive';
-import { InsertedSmartCardAuthApi } from '@votingworks/auth';
 import { Clock } from 'xstate/lib/interpreter';
+import { isReadyToScan } from '../../app_flow';
 import { interpret } from '../../interpret';
-import { Workspace } from '../../util/workspace';
-import { rootDebug } from '../../util/debug';
 import {
   InterpretationResult,
   PrecinctScannerError,
   PrecinctScannerMachineStatus,
   PrecinctScannerStateMachine,
 } from '../../types';
+import { rootDebug } from '../../util/debug';
+import { Workspace } from '../../util/workspace';
 import { recordAcceptedSheet, recordRejectedSheet } from '../shared';
-import { isReadyToScan } from '../../app_flow';
 
 const debug = rootDebug.extend('state-machine');
 
+let scanAndInterpretTimer: Timer | undefined;
```

Comment:
Just making sure you want to leave the timer in

Reply:
I do. I think we're going to want to be able to get this timing information in the future.